### PR TITLE
 [Project Addition] : Chest X-ray - Tuberculosis Disease Prediction 

### PR DESCRIPTION
**Pull request by aaradhyasinghgaur**
# Pull Request for DL-Simplified 💡

## Issue Title : Chest X-ray - Tuberculosis Disease Prediction 

- **Info about the related issue (Aim of the project)** : To predict and classify between Normal x-ray and tuberculosis diseases using chest X-ray images, a convolutional neural network (CNN) can be employed. This deep learning model will process the X-ray images to identify characteristic features and patterns indicative of each disease. The trained model will output a probability score for each disease, allowing for accurate and efficient diagnosis based on the visual data contained in the chest X-rays.
- **Name:**  Aaradhya Singh 
- **GitHub ID:** https://github.com/aaradhyasinghgaur
- **Email ID:** aaradhyasinghgaur@gmail.com 
- **Idenitfy yourself: (Mention in which program you are contributing in. Eg. For a JWOC 2022 participant it's, `JWOC Participant`)**  GSSOC-2024 Contributor

Closes: #755 

### Describe the add-ons or changes you've made 📃

1. Data Preparation: Collect and preprocess a balanced dataset of real and fake face images, including normalization, resizing, and augmentation.
2. Base Model Selection: EfficientNetB0,VGG16 , Xception , InceptionV3 like 5 different models excluding its top layers, to leverage its learned features.
3. Model Construction: Add custom layers on top of the base model for binary classification, compiling with appropriate loss and metrics.
4. Initial Training: Train the model with the base layers frozen to only update the new layers.
5. Fine-Tuning: Unfreeze some or all of the base model layers and continue training with a lower learning rate to fine-tune the entire network.
6.) EDA analysis.
7.) Comaprioson using performance matrices such as accuracy scores , confusion matrix etc.

## Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? ⚙️

Describe how it has been tested
Describe how have you verified the changes made

## Checklist: ☑️

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Comment by aaradhyasinghgaur** [comment]
@abhisheks008 kindly review my pr and assign me new issue if possible . Thank you

**Review by abhisheks008 — APPROVED** [review]
Approved @aaradhyasinghgaur
